k8s/cms-transcode-consumer/deployment.yaml

```
@@ -0,0 +1,25 @@
+      app: cms-transcode-consumer
+  template:
+    metadata:
+      labels:
+        app: cms-transcode-consumer
+    spec:
+      containers:
+      - name: cms-transcode-consumer
+        image: ghcr.io/callein/cms-transcode-consumer:latest
+        command: ["python", "src/consumer/request_consumer.py"]
+        envFrom:
+          - secretRef:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)